spec/models/account_spec.rb

```diff
@@ -596,11 +596,23 @@
         expect(account).to model_have_error_on_field(:display_name)
       end
 
+      it 'display name with newline and carriage returns gets modified to have only newline' do
+        account = Fabricate.build(:account, display_name: "My display name is\r\ncool")
+        account.valid?
+        expect(account.display_name).to eq "My display name is\ncool"
+      end
+
       it 'is invalid if the note is longer than 160 characters' do
         account = Fabricate.build(:account, note: Faker::Lorem.characters(161))
         account.valid?
         expect(account).to model_have_error_on_field(:note)
       end
+
+      it 'Note with newline and carriage return returns correct size' do
```

Comment:
Sorry, I fixed the name in the display name test, but not in this one, could you maybe match it to the newline one?